### PR TITLE
Updateserver XML for 5.2.0

### PR DESCRIPTION
**Pull request by Hackwar**
What the title says

### CODE REVIEW

**Comment by tecpromotion** [comment]
5.1.3-dev and 5.1.4-dev parts looks not correct to me.

File: www/core/nightlies/next_minor_extension.xml [hunk 1]
```diff
@@ -17,17 +17,17 @@
 		<php_minimum>8.1.0</php_minimum>
 		<maintainer>Joomla! Production Department</maintainer>
 		<maintainerurl>https://www.joomla.org</maintainerurl>
-		<targetplatform name="joomla" version="4.4.[67]" />
+		<targetplatform name="joomla" version=4.4.(([9])|(1[0]))" />
```
Comment (richard67) [inline www/core/nightlies/next_minor_extension.xml:20]:
```suggestion
		<targetplatform name="joomla" version="4.4.((9)|(1[0]))" />
```
Quote missing and single number simplified.

File: www/core/nightlies/next_patch_extension.xml [hunk 2]
```diff
@@ -94,17 +94,17 @@
 		<php_minimum>8.1.0</php_minimum>
 		<maintainer>Joomla! Production Department</maintainer>
 		<maintainerurl>https://www.joomla.org</maintainerurl>
-		<targetplatform name="joomla" version="4.4.[678]" />
+		<targetplatform name="joomla" version="4.4.(([9])|(1[0]))" />
```
Comment (richard67) [inline www/core/nightlies/next_patch_extension.xml:97]:
```suggestion
		<targetplatform name="joomla" version="4.4.(9|(1[0]))" />
```
Single number simplified.

File: www/core/nightlies/next_minor_extension.xml (hunk 1 above)
Comment (richard67) [inline www/core/nightlies/next_minor_extension.xml:20]:
```suggestion
		<targetplatform name="joomla" version="4.4.(9|(1[0]))" />
```
Quote missing and simplify single number.

File: www/core/nightlies/next_patch_extension.xml (hunk 2 above)
Comment (MacJoom) [reply www/core/nightlies/next_patch_extension.xml]: Thanks

**Pull request merged**